cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1669,7 +1669,8 @@ bool SubstraitToVeloxPlanConverter::canPushdownOr(
   }
```

Comment:
And I suggest we add a `setOrRange` function in RangeRecorder which returns false when other conditions already exist for a field.

Reply:
`setInRange` itself retains a parameter `forOrRelation`, which is used for verification. We only need to judge the first singularOrList in or.

Reply:
Thanks for your suggestion, updated.